sections/dashboard/Stake/EscrowTable.tsx

```diff
@@ -237,15 +227,15 @@ const EscrowTable = () => {
 							{t('dashboard.stake.tabs.stake-table.kwenta-token')}
 						</div>
 					</div>
-					<VestButton disabled={!vest} onClick={() => setOpenConfirmModal(true)}>
+					<VestButton disabled={false} onClick={openConfirmModal}>
```

Comment:
can just omit instead of false

Reply:
Yeah. This was a placeholder anyway. Updated. That said, I think omitting sets it to `true` though. I believe it's done that way since most booleans default to false.

Reply:
Ah I just meant to remove the disabled prop rather than set to false but yea all good now anyway.